azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/redux/middleware/handler/CallingMiddlewareActionHandler.kt

```diff
@@ -320,15 +354,15 @@ internal class CallingMiddlewareActionHandlerImpl(
 
     private fun subscribeCamerasCountUpdate(store: Store<ReduxState>) {
         coroutineScope.launch {
-            callingService.getCamerasCountStateFlow().collect {
+            callingService.getCamerasCountStateFlow()?.collect {
```

Comment:
Why do we need "?" null check here (and five more places in the file)? Returned type by callingService.getCamerasCountStateFlow() is not nullable